code-style-guide.md

```
@@ -0,0 +1,2451 @@
+```swift
+public class MyViewController: UIViewController {
+  @IBOutlet private var tableView: UITableView!
+}
+
+@Observable class MySecondViewController {
+    // ...
+}
+```
+
+# 5. Naming
+### 5.1 Apple’s API Style Guidelines
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)